tests/tests_geomstats/test_geometry/data/positive_lower_triangular_matrices.py

```diff
@@ -3,33 +3,12 @@
 import geomstats.backend as gs
 from geomstats.test.data import TestData
 
-from .base import OpenSetTestData
 from .riemannian_metric import RiemannianMetricTestData
 
 EULER = gs.exp(1.0)
 SQRT_2 = math.sqrt(2)
 
 
-class PositiveLowerTriangularMatricesTestData(OpenSetTestData):
-    def gram_vec_test_data(self):
-        return self.generate_vec_data()
-
-    def differential_gram_vec_test_data(self):
-        return self.generate_vec_data()
-
-    def inverse_differential_gram_vec_test_data(self):
-        return self.generate_vec_data()
-
-    def gram_belongs_to_spd_matrices_test_data(self):
-        return self.generate_random_data()
-
-    def differential_gram_belongs_to_symmetric_matrices_test_data(self):
-        return self.generate_random_data()
-
-    def inverse_differential_gram_belongs_to_lower_triangular_matrices_test_data(self):
-        return self.generate_random_data()
-
-
 class PositiveLowerTriangularMatrices2TestData(TestData):
```

Comment:
Maybe remove the `2` in the PositiveLowerTriangularMatrices2 here.

Reply:
The 2 here means `n=2`, as this corresponds to smoke data on a particular space.